contracts/rateproducer/rateproducer.cpp

```diff
@@ -1,287 +0,0 @@
-#include <eosio/eosio.hpp>
```

Comment:
@kriskoin  lets not delete this file

Reply:
@xavier506  so sorry it was a mistake, fixed !!

Reply:
thanks!